CODE_OF_CONDUCT.md

```diff
@@ -1,46 +1,119 @@
-# Contributor Covenant Code of Conduct
+# Our Pledge
+In the interest of fostering an open, collaborative, and welcoming environment, we as contributors and maintainers of nf-core, pledge to making participation in our projects and community a harassment-free experience for everyone, regardless of:
 
-## Our Pledge
+- Age
+- Body size
+- Familial status
+- Gender identity and expression
+- Geographical location
+- Level of experience
+- Nationality and national origins
+- Native language 
+- Physical and neurological ability
+- Race or ethnicity
+- Religion
+- Sexual identity and orientation
+- Socioeconomic status
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+Please note that the list above is alphabetised and is therefore not ranked in any order of preference or importance. 
 
-## Our Standards
+# Code of Conduct at nf-core (v1.0) 
+#### Date: March 12, 2021
 
-Examples of behavior that contributes to creating a positive environment include:
+## Preamble
 
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy towards other community members
+**Note: This Code of Conduct (CoC) has been drafted by the nf-core Safety Officer and been edited after input from members of the nf-core team and others. "We", in this document, refers to the Safety Officer and members of the nf-core core team, both of whom are deemed to be members of the nf-core community and are therefore required to abide by this Code of Conduct. This document will be amended periodically to keep it up-to-date and the in case of any dispute, the most current version.**
```

Comment:
Just noticed a typo, the last sentence should read "This document will amended periodically to keep it up-to-date, and in case of any dispute, the most current version will apply."

Reply:
```suggestion
> Note: This Code of Conduct (CoC) has been drafted by the nf-core Safety Officer and been edited after input from members of the nf-core team and others. "We", in this document, refers to the Safety Officer and members of the nf-core core team, both of whom are deemed to be members of the nf-core community and are therefore required to abide by this Code of Conduct. This document will amended periodically to keep it up-to-date, and in case of any dispute, the most current version will apply.
```

Reply:
Thank you!